doc/user_doc/vic_installation/vch_installer_examples.md

```diff
@@ -21,16 +21,15 @@ You can install vSphere Integrated Containers directly on an ESXi host that is n
 <pre>vic-machine<i>-darwin</i><i>-linux</i><i>-windows</i> create
 --target <i>esxi_host_IPv4_address_or_FQDN</i>
 --user root
---compute-resource /ha-datacenter/host/localhost.eng.vmware.com/Resources/
 --image-datastore <i>datastore_name</i>
 </pre>
 
 <a name="cluster"></a>
 ## Deploy a Virtual Container Host in a vCenter Server Cluster ##
 
-If vCenter Server manages clusters, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
+If vCenter Server manages more than one cluster, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
```

Comment:
Actually, here we did enhancement to make it easier.
So relative path is accepted, for example, you can specify path as host/Resources/path/somewhere.
Or in ESXi, you could just write the path after Resources, for example path/somewhere
In VC, that could be 'vc cluster'/path/somewhere